src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/ApiVersionHelper.cs

```diff
@@ -112,7 +112,7 @@ private static string[] GetApiVersionsForResourceType(AzureContext context, stri
                 getFirstPage: () => resourceManagerClient
                     .ListObjectColleciton<ResourceProviderDefinition>(
                         resourceCollectionId: resourceCollectionId,
-                        apiVersion: Constants.DefaultApiVersion,
+                        apiVersion: Constants.ProvidersApiVersion,
```

Comment:
also, if the default api version is no longer used, can you remove it from constants.cs?